WalletWasabi/Crypto/SchnorrBlindingSignature.cs

```diff
@@ -43,7 +43,7 @@ public uint256 BlindMessage(uint256 message, PubKey rpubkey, PubKey signerPubKey
 				var R = rECPubKey.Q.ToGroupElementJacobian();
 				var t = FE.Zero;
 
-				retry:
+			retry:
 
 				RandomUtils.GetBytes(tmp);
 				_v = new Scalar(tmp, out int overflow);
```

Comment:
Here it is OK?

Reply:
Yes because `v` is hidden behing the DL and after that it is hashed so, it provides no clues to the coordinator. 
Anyway, this comment of yours made me discover another problem that I will fix now.